src/Components/IncidentList/IncidentList.Component.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import {
+    Table,
+    TableBody,
+    TableHeader,
+    TableHeaderColumn,
+    TableRow,
+    TableRowColumn,
+} from 'material-ui/Table';
+import FlatButton from 'material-ui/FlatButton';
+import PropTypes from 'prop-types';
+
+//styling
+import './IncidentList.scss';
+
+/**
+ * @class IncidentList
+ */
+export default class IncidentList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: [],
+      flags: {
+        green: '/assets/images/green_flag.svg',
+        yellow: '/assets/images/yellow_flag.svg',
+        red: '/assets/images/red_flag.svg'
+      }
+    };
+  }
+
+  /**
+   * Method to determine if a row is selected
+   * @param {number} index - Row index
+   * @returns {boolean} 
+   */
+  isSelected = (index) => {
+    return this.state.selected.indexOf(index) !== -1;
+  };
+
+  /**
+   * Method to set selected row(s)
+   * @param {number} selectedRows - Index of selected Row(s)
+   */
+  handleRowSelection = (selectedRows) => {
+    this.setState({
+        selected: selectedRows,
+    });
+  };
+
+  /**
+   * Method to get incident flag Icon
+   * @param {string} flagType - incident flag type
+   * @returns {string} flagType URL
+   */
+  getIncidentFlag = (flagType) => {
+    if (flagType === 'type A') {
+      return this.state.flags.green;
+    } else if (flagType === 'type B') {
+      return this.state.flags.yellow;
+    } else {
+      return this.state.flags.red;
+    }
+  }
+  
+  render() {
+
+    const { incidents } = this.props;
+
```

Comment:
unnecessary line spaces

Reply:
It's important to separate the purpose of each code block. esp. return & data